tubesync/sync/views.py

```diff
@@ -296,23 +296,30 @@ class EditSourceMixin:
     def form_valid(self, form: Form):
         # Perform extra validation to make sure the media_format is valid
         obj = form.save(commit=False)
-        source_type = form.cleaned_data['media_format']
+        # temporarily use media_format from the form
+        saved_media_format = obj.media_format
+        obj.media_format = form.cleaned_data['media_format']
```

Comment:
I only took a guess at what was intended here.

Reply:
Given `source_type` isn't used in that method from what I can see I'd go with a copy/paste error that didn't break anything so went unnoticed.